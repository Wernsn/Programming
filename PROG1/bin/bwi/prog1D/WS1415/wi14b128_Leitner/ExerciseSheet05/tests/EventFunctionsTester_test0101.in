1
a very very very very long title
2012
12
31
55
1
title
1581
2200
1582
1
13
10
1
15
16
0
0

