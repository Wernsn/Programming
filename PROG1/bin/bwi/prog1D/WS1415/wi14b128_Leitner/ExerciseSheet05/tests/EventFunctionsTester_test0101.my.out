scanEvent: enter title: enter year (1582-2199): enter month (1-12): enter day (1-31): enter number of attendees: |a very very very very long title|Monday, December 31st 2012|55||
scanEvent: enter title: enter year (1582-2199): too low!too high!enter month (10-12): too low!too high!enter day (16-31): too low!too low!enter number of attendees: |title|Saturday, October 16th 1582|0||
